src/clusterfuzz/_internal/cron/project_setup.py

```diff
@@ -864,6 +885,17 @@ def _sync_job(self, project, info, corpus_bucket_name, quarantine_bucket_name,
       file_github_issue = info.get('file_github_issue', False)
       job.environment_string += f'FILE_GITHUB_ISSUE = {file_github_issue}\n'
 
+      # Android creates device-specific queues during project setup.
+      queue_id = info.get('queue_id', False)
```

Comment:
see my comment below and making this:

```
'platform': 'ANDROID'
'queue_id': 'pixel7'
```

instead. 

Then we can key on 'platform'  == 'ANDROID' for many of thjese android specific pieces, and avoid making assumptions on formatting (i.e. the splitting on -). We can also set 'ANDROID_X86' in the projects.json without adding a special case here.

Reply:
Done.